share/doc/homebrew/Maintainer-Guidelines.md

```diff
@@ -6,20 +6,118 @@ definitely not a beginner’s guide.
 
 Maybe you were looking for the [Formula Cookbook](Formula-Cookbook.md)?
 
+----
+
+Quick links:
+  [General Guidelines](#general-guidelines) •
+  [Core and Core PRs](#core-and-core-prs) •
```

Comment:
I'd just go for a bullet list here. My only concern is that stuff like this tends to become outdated when done manually.